unikraft/app/application.go

```diff
@@ -1030,3 +1033,12 @@ func (app application) Save(ctx context.Context) error {
 func (app application) Volumes() []*volume.VolumeConfig {
 	return app.volumes
 }
+
+func (app application) AddLibrary(ctx context.Context, library lib.LibraryConfig) error {
+	if app.libraries == nil {
+		app.libraries = make(map[string]*lib.LibraryConfig)
```

Comment:
Instead of replacing all instances of `make` with `kraftMake`, simply instantiate the structure like so:
```go
if app.libraries == nil {
  app.libraries = map[string]*lib.LibraryConfig{}
}
```

Reply:
Done 👍🏽